### PR TITLE
MCMC

### PR DESCRIPTION
**Pull request by evahunter**
Current version of MCMC: for some reason still getting module error finding emcee even though it is added to requirements - unsure why.

### CODE REVIEW

**Comment by aarmey** [comment]
@evahunter you have to make sure you are running your notebook inside of the virtual environment. Run `make venv`, and then you can change your kernel to be the Python version at `./venv/bin/python3`

**Comment by aarmey** [comment]
Let me know if this fixes things.

File: emceetest.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    return log_probability
+
+ndim, nwalkers = 24, 100
+p0 = np.random.randn(nwalkers, ndim)
+
+sampler = emcee.EnsembleSampler(nwalkers, ndim, log_prob, args = [A_antiD, mean_3a])
+sampler.run_mcmc(p0, 10000)
+
+
+# Traceback (most recent call last):
```
Comment (aarmey) [inline emceetest.py:38]: It looks like someone managed to run this within a virtual environment, though... why the notebook in that case?

Comment (evahunter) [reply emceetest.py:38]: I had the notebook first and I was having problems with it so I made it into a .py and was able to run it. I think initially my VS was having problems installing ipykernel even though I tried a bunch of different ways.

Comment (aarmey) [reply emceetest.py:38]: Ok, then is there some feedback you want on this code?

Comment (evahunter) [reply emceetest.py:38]: We were hoping you could look it over and make sure this is correct/what you wanted for the MCMC code. Thanks.

**Comment by aarmey** [comment]
Made some small adjustments. Notice how I added a term to push the inferred numbers to be positive. I think we can also pull the normal distribution std from the data to fix the model a bit more (can help with this later).

Play with this some and make sure you are following how it works. Making some of the plots from the emcee documentation could be helpful. From there, it would be helpful to add all the other activities (we'll infer them all at once).

**Pull request closed without merging**